plug.vim

```diff
@@ -935,7 +940,8 @@ function! s:log(bullet, name, lines)
     else
       let pos = 4
     endif
-    call append(pos - 1, s:format_message(a:bullet, a:name, a:lines))
+    let filtered = substitute(a:lines, '[[:cntrl:]]', '', 'g')
```

Comment:
1) When I dump out the contents of a:lines before filtered what I see is most lines end in `\r\n`, while the first few oddly end in `\n\n`. I'm not sure why we are getting Windows EOL when I'm on Linux. So the [[:cntrl:]] could be replaced with [\r\n] if we want to be exact. 

2) Since I don't know what neovim was doing before, can't comment yet. Did older versions only have 1 newline? Trimmed? Perhaps in a recent commit they changed the EOL for some other feature. Might build some older commits to see.

3) Well as a safety issue we could leave it, when using append you aren't ever supposed to leave any new line control characters in the buffer. Got caught by that on Python a few times.

Also this new EOL policy broke s:lines. Quite a nuisance.